### ISSUE TITLE
Fatal error in the FlashHelper using Cake 3.1

### ISSUE THREAD

**Issue by burzum** [FriendsOfCake/bootstrap-ui#47]
I've updated my composer today and got these versions:

```
  - Updating cakephp/cakephp dev-master (1014b09 => 3a3de18)
    Checking out 3a3de18ca3ed7da0fe16daeee186281e7c033309

  - Updating friendsofcake/bootstrap-ui dev-master (7cc5142 => 3c87f0f)
    Checking out 3c87f0f9e187a6c93b584e6ca13e37e2537c974b
```

I've started getting a fatal error line 55 from the FlashHelper then: `unsupported opperand type line 55` The line in question is this:

```php
$flash['params'] += $this->_config;
```

I've started debugging this and found out that $flash is now an array of flash messages. I'm not sure how to fix it properly. My guess is that a foreach is needed and the config needs to be merged for reach of the flashes?

**Comment by davidyell** [comment]
What about `array_merge_recursive()` ?

**Comment by dereuromark** [comment]
Related to https://github.com/FriendsOfCake/bootstrap-ui/issues/43

**Comment by burzum** [comment]
This fixes it for me but I have no idea if this is the right way to fix it.

```php
    public function render($key = 'flash', array $options = [])
    {
		if (!$this->request->session()->check("Flash.$key")) {
			return null;
		}

		$flash = $this->request->session()->read("Flash.$key");
		if (!is_array($flash)) {
			throw new \UnexpectedValueException(sprintf(
				'Value for flash setting key "%s" must be an array.',
				$key
			));
		}

		$output = '';
		$flashes = $flash;
		foreach ($flashes as $flash) {
			$flash = $options + $flash;
			$flash['params'] += $this->_config;
			$this->request->session()->delete("Flash.$key");

			$element = $flash['element'];
			if (strpos($element, '.') === false &&
				preg_match('#Flash/(default|success|error|info|warning)$#', $element, $matches)
			) {
				$class = $matches[1];
				$class = str_replace(['default', 'error'], ['info', 'danger'], $class);

				if (is_array($flash['params']['class'])) {
					$flash['params']['class'][] = 'alert-' . $class;
				}
				$element = 'BootstrapUI.Flash/default';
			}

			$output .= $this->_View->element($element, $flash);
		}
		return $output;
    }
```

**Comment by ADmad** [comment]
Fixed in #48
